[2021-11-29 12:17:56,403] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_3 2020-01-04T00:00:00+00:00 [queued]>
[2021-11-29 12:17:56,832] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_3 2020-01-04T00:00:00+00:00 [queued]>
[2021-11-29 12:17:56,833] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-11-29 12:17:56,834] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-11-29 12:17:56,834] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-11-29 12:17:57,188] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_3> on 2020-01-04T00:00:00+00:00
[2021-11-29 12:17:57,289] {standard_task_runner.py:52} INFO - Started process 107017 to run task
[2021-11-29 12:17:57,368] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_3', '2020-01-04T00:00:00+00:00', '--job-id', '30', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmp537v7fge', '--error-file', '/tmp/tmpjdmneg_p']
[2021-11-29 12:17:57,599] {standard_task_runner.py:77} INFO - Job 30: Subtask task_3
[2021-11-29 12:17:59,158] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_3 2020-01-04T00:00:00+00:00 [running]> on host airflowvm
[2021-11-29 12:18:00,826] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_3
AIRFLOW_CTX_EXECUTION_DATE=2020-01-04T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-04T00:00:00+00:00
[2021-11-29 12:18:00,827] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-11-29 12:18:00,827] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-11-29 12:18:00,941] {subprocess.py:75} INFO - Output:
[2021-11-29 12:18:04,158] {subprocess.py:83} INFO - Command exited with return code 0
[2021-11-29 12:18:04,831] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_3, execution_date=20200104T000000, start_date=20211129T121756, end_date=20211129T121804
[2021-11-29 12:18:06,943] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-11-29 12:18:07,019] {local_task_job.py:151} INFO - Task exited with return code 0
